src/DynamoCore/DynamoCore.csproj

```diff
@@ -395,11 +396,14 @@ limitations under the License.
     <Copy SourceFiles="@(ExternProtoGeometryXml)" DestinationFolder="$(OutputPath)\$(UICulture)" />
     <Copy SourceFiles="@(ExternProtoGeometryUICulture)" DestinationFolder="$(OutputPath)\$(UICulture)" />
     <Copy SourceFiles="@(ExternProtoGeometryLibGLocale)" DestinationFolder="$(OutputPath)\libg_locale\%(RecursiveDir)" />
-    <Copy SourceFiles="@(ExternLibG220)" DestinationFolder="$(OutputPath)libg_220\" />
-    <Copy SourceFiles="@(ExternLibG221)" DestinationFolder="$(OutputPath)libg_221\" />
-    <Copy SourceFiles="@(ExternLibG222)" DestinationFolder="$(OutputPath)libg_222\" />
-    <Copy SourceFiles="@(ExternLibG223)" DestinationFolder="$(OutputPath)libg_223\" />
-    <Copy SourceFiles="@(ExternLibG224)" DestinationFolder="$(OutputPath)libg_224\" />
+    
+    <Copy SourceFiles="@(ExternLibG220)" DestinationFolder="$(OutputPath)libg_220_0_0\" />
+    <Copy SourceFiles="@(ExternLibG221)" DestinationFolder="$(OutputPath)libg_221_0_0\" />
```

Comment:
Why don't we just get rid of 220 and 221?

Reply:
briefly spoke about this - we'll keep these for now for flexibility, but don't intend on updating these versions any longer.